src/main/resources/data/1817/CompanyManager.xml

```diff
@@ -0,0 +1,237 @@
+				<SpecialBaseTokenLay connected="yes" extra="no" free="yes"/>
+		</SpecialProperties>
+	</Company>
+	<Company name="CMinor" type="Private" basePrice="30" revenue="0"
+		longname="Minor Coal Mine">
+		<SpecialProperties>
+					<SpecialProperty condition="ifOwnedByCompany" when="tileLayingStep" class="net.sf.rails.game.special.SpecialTileLay">
+				<SpecialTileLay location="B25,C20,C24,E18,F15,G12,G14,H11,H13,H15,I8,I10" tile="3,4,58" extra="no" free="yes"/>
+				
+		</SpecialProperties>
+	</Company>
+		</Company>
+	<Company name="CMedium" type="Private" basePrice="60" revenue="0"
+		longname="Medium Coal Mine">
+		<SpecialProperties>
+					<SpecialProperty condition="ifOwnedByCompany" when="tileLayingStep" class="net.sf.rails.game.special.SpecialTileLay">
+				<SpecialTileLay location="B25,C20,C24,E18,F15,G12,G14,H11,H13,H15,I8,I10" tile="3,4,58" extra="no" free="yes"/>
+					<SpecialProperty condition="ifOwnedByCompany" when="tileLayingStep" class="net.sf.rails.game.special.SpecialTileLay">
+				<SpecialTileLay location="B25,C20,C24,E18,F15,G12,G14,H11,H13,H15,I8,I10" tile="3,4,58" extra="no" free="yes"/>
+		</SpecialProperties>
+	</Company>
+		</Company>
+	<Company name="CMajor" type="Private" basePrice="90" revenue="0"
+		longname="Major Coal Mine">
+		<SpecialProperties>
+					<SpecialProperty condition="ifOwnedByCompany" when="tileLayingStep" class="net.sf.rails.game.special.SpecialTileLay">
+				<SpecialTileLay location="B25,C20,C24,E18,F15,G12,G14,H11,H13,H15,I8,I10" tile="3,4,58" extra="no" free="yes"/>
+					<SpecialProperty condition="ifOwnedByCompany" when="tileLayingStep" class="net.sf.rails.game.special.SpecialTileLay">
+				<SpecialTileLay location="B25,C20,C24,E18,F15,G12,G14,H11,H13,H15,I8,I10" tile="3,4,58" extra="no" free="yes"/>
+					<SpecialProperty condition="ifOwnedByCompany" when="tileLayingStep" class="net.sf.rails.game.special.SpecialTileLay">
+				<SpecialTileLay location="B25,C20,C24,E18,F15,G12,G14,H11,H13,H15,I8,I10" tile="3,4,58" extra="no" free="yes"/>
+		</SpecialProperties>
+	</Company>
+	
+	<!-- Note two supported colour specification formats:
+	     RGB decimal with commas and RGB hexadecimal without commas -->
+	<Company name="B&amp;M" type="Public" tokens="2" fgColour="000000" bgColour="60E060"
+			longname="Boston &amp; Maine">
+		<Home hex="E23"/>
+	</Company>	     
+	<Company name="B&amp;O" type="Public" tokens="3" fgColour="255,255,255" bgColour="0,0,255"
+			longname="Baltimore &amp; Ohio">
+		<FirstTrainCloses type="Private" name="B&amp;O"/>
+		<Home hex="I15"/>
+	</Company>
+	<Company name="C&amp;O" type="Public" tokens="3" fgColour="000000" bgColour="A0E0FF"
+			longname="Chesapeake &amp; Ohio">
+		<IfOption name="Variant" value="Basegame,Pere Marquette,Reading,Simple,Wabash">
+			<Home hex="F6"/>
+		</IfOption>
+		<IfOption name="Variant" value="Coalfields,Coalfields&amp;Reading">
+			<Home hex="K13"/>
+		</IfOption>
```

Comment:
These variants are from 1830 and don't exist in 1817.

The whole list of companies isn't the 20 companies that are in 1817.